includes/uiwidgets-ref-dateSerializationFormat.md

```diff
@@ -0,0 +1,17 @@
+Specify this property in the following cases:
+
+- The [dataSource](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/dataSource.md '{basewidgetpath}/Configuration/#dataSource') is empty or not set at design time. The **dateSerializationFormat** is needed because the {WidgetName} cannot detect it automatically without a data source.
```

Comment:
```suggestion
- The [dataSource](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/dataSource.md '{basewidgetpath}/Configuration/#dataSource') is empty or not set at design time. The **dateSerializationFormat** is needed, because the {WidgetName} cannot detect it automatically without a data source.
```